dojo/fixtures/test_type.json

```diff
@@ -726,5 +726,12 @@
         },
         "model": "dojo.test_type",
         "pk": 195
+    },
+    {
+        "fields": {
+            "name": "Cloudsploit"
+        },
+        "model": "dojo.test_type",
+        "pk": 196
```

Comment:
Modification of this file is no longer required

Reply:
change reverted.